flotiq/flotiq-codegen-ts
# #24629 add flag for instant js compilation

**Added:**
- flag to compiling flotiqApi to js 

**Testing:**
- you have to have runing docker engine to run this command 
- Install from branch `npm install git+https://github.com/flotiq/flotiq-codegen-ts.git#feature/24629-add-flag-for-instant-js-compilation`
- Run `npx flotiq-codegen-ts generate`

**If you wish to get API compiled in js use:** 

`npx flotiq-codegen-ts generate --copiled-js`

**Testing procedure**
- run `npx flotiq-codegen-ts generate --copiled-js` and check if `flotiqApi` directory was created with `.js` and `.d.ts` files inside 
- delete `flotiqApi` directory
- run `npx flotiq-codegen-ts generate` and check if `flotiqApi` directory was created with `.ts` files
- delete `flotiqApi` directory
-  run `npx flotiq-codegen-ts generate --copiled-js` then import `flotiqApi` in project and check if `autocompletion` works
- and new `ctd` to your account, then rerun `npx flotiq-codegen-ts generate --copiled-js` and check if changes are new `ctd` is visible in `autocompletion`
